pkg/controller/common/terraform.go

```diff
@@ -55,6 +55,8 @@ func NewTerraformer(factory terraformer.Factory, config *rest.Config, purpose st
 	}
 
 	return tf.
+		UseV2(true).
+		SetLogLevel("debug").
```

Comment:
You can also choose the `info` level for terraformer. I only used `debug` for gcp as it was the first provider extension, for which we enabled v2. I will change it back to `info` soon.
But you can of course also use `debug` as a first step, but it should also be changed to `info` afterwards to not flood the logging stack :)